test/test_3_extruders_with_runout_sensors/test_all.cpp

```diff
@@ -0,0 +1,31 @@
+/* START_CONFIGURATION
+
+opt_set EXTRUDERS 3;
+opt_set TEMP_SENSOR_1 1;
+opt_set TEMP_SENSOR_2 1;
+opt_set TEMP_2_PIN 4;
+opt_set TEMP_3_PIN 4;
+opt_set HEATER_2_PIN 4;
+opt_set E2_STEP_PIN 4;
+opt_set E2_DIR_PIN 4;
+opt_set E2_ENABLE_PIN 4;
+opt_set E3_STEP_PIN 4;
+opt_set E3_DIR_PIN 4;
+opt_set E3_ENABLE_PIN 4;
+opt_set NUM_RUNOUT_SENSORS 3;
+opt_enable FILAMENT_RUNOUT_SENSOR;
+opt_set FIL_RUNOUT_PIN 4;
+opt_set FIL_RUNOUT2_PIN 4;
+opt_set FIL_RUNOUT3_PIN 4;
+opt_set FILAMENT_RUNOUT_SCRIPT "\"M600 %c\"";
+opt_enable ADVANCED_PAUSE_FEATURE;
+opt_enable EMERGENCY_PARSER;
+opt_enable NOZZLE_PARK_FEATURE;
```

Comment:
It may be cleaner to replace these with INI files based on `config.ini`, which can also download example configs and do some other tricks. To get the `config.ini` to apply to the build it just has to be copied into `Marlin/`.

Reply:
I thought about that, but wasn't familiar with how the config.ini worked in order to try it out.

I hesitated to completely remove the .cpp files because PlatformIO might not see these as tests work building anymore. I can give it a try.

Reply:
I have this working now, I'm just testing the workflow behavior now.